src/hooks/useAccountCalls.ts

```diff
@@ -0,0 +1,22 @@
+import { FetchOptions } from "../model/fetchOptions";
+import { getCallerArchive } from "../services/archiveRegistryService";
+import { getCallsByAccount } from "../services/callsService";
+import { ExtrinsicsOrder } from "../services/extrinsicsService";
+
+import { usePaginatedResource } from "./usePaginatedResource";
+
+export function useAccountCalls(
+	network: string,
+	address: string,
+	order?: ExtrinsicsOrder,
+	options?: FetchOptions
+) {
+	const filter = {
+		callerPublicKey_eq: address,
+	};
+	if (getCallerArchive(network)){
```

Comment:
tohle s hookama nesmis delat, pak to bude delat kraviny, hook musi byt volany vzdy v top level vstve tela funkce, tedy zadny podnimky, cykly apod.

proc to nedelas stejne jako pro extrinsicy?